README.md

```diff
@@ -117,10 +88,8 @@ to ensure the final objects can resolve each others.
 
 ## Requirements
```

Comment:
I don't know if this is the right place to add it, but we should note into this README that the generated code must be consumed with tinygo 0.28.1 or later, otherwise there will be panics at runtime.

Reply:
Good point. There seems to exist build tags such as  `+build tinygo` , but haven't found if there is a versioned one. Worth investigating I would say.

If not, a simple Makefile check could be good too.

Reply:
I don't think they do have such a build flag (a versioned one I mean)

Reply:
I was referring to [this one](https://github.com/kubewarden/k8s-objects-generator/blob/main/Makefile), but please ignore my suggestion, as this is about the generated code and not k8s-objects-generator binary.